### DISCUSSION TITLE
Como incluir em um projeto flutter um .jar e libs.so

### DISCUSSION THREAD

**Discussion by Gabrielvp** [Flutterando/forum#114] [Perguntas e Respostas]
Sou novo no flutter, na empresa que trabalho está sendo criado um totem de pagamento, o cliente vai no totem coloca o cpf aparece o débito e ele efetua o pagamento. Preciso integrar com sitef para fazer pagamentos via cartão, eles enviaram um .jar e libs.so para fazer a integração mas o manual enviado é para android nativo, como faço para utilizar no flutter?

**Comment by jonathanBAugusto** [comment]
Se não me falha a memoria o .jar é uma aplicação que abre um endpoint para receber um xml. verifica ai que provavelmente tu só vai precisar fazer um POST para o endereço/porta que o .jar vai abrir e o .jar faz o resto .
